test/test_Ids.py

```diff
@@ -21,3 +21,37 @@ def test_ids_ncbi(self):
         x.ncbi()
         assert len(x.ids) > 0
         assert x.db_ids == "ncbi"
+    
+    @vcr.use_cassette("test/vcr_cassettes/ids_gbif_1.yml")
+    @pytest.mark.skipif(
+        "TRAVIS" in os.environ and os.environ["TRAVIS"] == "true",
```

Comment:
no reason to skip these tests that I know of - the failing test was a different one AFAICT, and was b/c the env var stored on Travis is not allowed to be used on PR's - i'll fix that soon

Reply:
Good point! I will remove these flags for the gbif tests

Reply:
thanks